modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -29,13 +29,152 @@ image::api-analytics-detail-panel.png[]
 +
 You can click the *View API in API Manager* link to view API analytics in API Manager.
 
-== Requests
+== New Metrics View for API Analytics Dashboards
+
+API analytics dashboards include various pages to view different types of charts:
+
+[%header,cols="1,4"]
+|===
+| API Analytics Dashboards | Charts
+
+| *Overview* a|
+
+* Request Volume
+* Failed Requests
+* Policy Violations
+* Average Response Time
+
+| *Requests* a|
+
+* Requests
+* Requests by Method
+* Requests by Response Code
+* Requests by Request Size
+* Requests by Response Size
+
+| *Policies* a|
+
+* Requests
+* Policy Violations by Outcome
+* Policy Violations by Violation ID
+
+| *Failures* a|
+
+* Failures
+* Failures by Method
+* Requests by Response Code
+* Requests by Request Size
+* Requests by Response Size
+
+| *Performance* a|
+
+* Requests by Performance (line chart)
+* Requests by Performance (table)
+* Slow Requests (requires Titanium)
+
+| *Client Applications* a|
+
+* Requests by Client ID
+* Requests by Client IP
+* Requests by Country
+* Requests by State/Province
+
+|===
+
+=== Overview Charts
+
+The charts on the *Overview* dashboard are defined as follows:
+
+Request Volume:: The sum of requests in the chosen time period for the given API.
+
+Failed Requests:: The sum of failed requests in the chosen time period for the given API.
+
+Policy Violations:: The sum of requests for policy violations in the chosen time period, grouped by policy violation ID.
+
+Average Response Time:: The average response time of requests in the chosen time period for the given API.
+
+=== Requests Charts
+
+The charts in the *Requests* dashboard are defined as follows:
+
+Requests:: The sum of requests in the chosen time period, grouped by the top 100 paths.
+
+Requests by Method:: The sum of requests in the chosen time period, grouped by method.
+
+Requests by Response Code:: The sum of requests in the chosen time period, grouped by response code.
+
+Requests by Request Size:: The sum of requests in the chosen time period, grouped by request size.+
+
+Requests by Response Size:: The sum of requests in the chosen time period, grouped by response size.+
+
+=== Policies Charts
+
+The charts in the *Policies* dashboard are defined as follows:
+
+Requests:: The sum of requests of policy violations in the chosen time period, grouped by top N paths.
+
+// should N be 100? or is this undecided?
```

Comment:
@pgaliana Please advise on this?

Reply:
Default value for N is 100.
The value is configurable using the following property:
anypoint.platform.config.analytics.agent.api_metrics.path.sampling_size